test/hyphen.spec.js

```diff
@@ -0,0 +1,102 @@
+var chai = require('chai');
+var webdriver = require('selenium-webdriver');
+var helpers = require('scribe-test-harness/helpers');
+
+var expect = chai.expect;
+
+var when = helpers.when;
+var given = helpers.given;
+var givenContentOf = helpers.givenContentOf;
+var executeCommand = helpers.executeCommand;
+
+var scribeNode;
+
+var driver;
+before(function () {
+    driver = helpers.driver;
+});
+
+beforeEach(function() {
+    scribeNode = helpers.scribeNode;
+});
+
+function clickCleanup() {
+    scribeNode.click();
+    return executeCommand('cleanup');
+}
+
+describe('Hyphen cleaining', () => {
+    givenContentOf('<p>content -- </p>', () => {
+        when('the command is executed', () => {
+            beforeEach(clickCleanup);
+
+            it('should replace the -- with an endash', () => {
+                return scribeNode.getInnerHTML().then((innerHTML) => {
+                    expect(innerHTML).to.include('<p>content – </p>');
```

Comment:
Ironically if I have pasted in the wrong character then all my tests might be for nothing!